config/locales/translation_en.yml

```diff
@@ -183,64 +183,64 @@ en:
           page_title: Add Output to Elasticsearch
       out_mongo:
         option_guide: |
-          For each config parameter, please refer to the <a href="https://docs.fluentd.org/articles/out_mongo" target="_blank">MongoDB output plugin</a> documentation page.
-        show:
+          For each config parameter, please refer to the <a href="https://docs.fluentd.org/output/mongo" target="_blank">MongoDB output plugin</a> documentation page.
+          show:
```

Comment:
invalid indent?

Reply:
Oops, I'm sorry for my careless.
I've fixed it.